ee/clickhouse/demo.py

```diff
@@ -1,8 +1,8 @@
 from typing import Dict, List
 from uuid import uuid4
 
-from ee.clickhouse.models.event import create_event
-from ee.clickhouse.models.session_recording_event import create_session_recording_event
+from posthog.models.event.util import create_event
+from posthog.models.session_recording_event.util import create_session_recording_event
```

Comment:
come back here after finding the file to see if there's a new util folder or not